CMakeLists.txt

```diff
@@ -88,6 +88,8 @@ if(WITH_SANITIZERS)
     link_libraries(asan ubsan)
 endif()
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g ${SANITIZER_FLAGS}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
```

Comment:
Do we really want these changes in this PR? If so, please do so in a separate commit.